docs/UserGuide.md

```diff
@@ -236,6 +236,45 @@ Perhaps for some reason, you decide not to go through with an application. Maybe
 
 :information_source: Note the successful command execution message in the system feedback box upon successful deletion of the company.
 
+## Use Case #3: Editing a company or a role <a id="usecase3"></a>
+Say you received an email for an interview and the point of contact's email is different.
+Here's how you can make changes to the application:
+1. [Edit the company](#c-edit-c) from the company list.  
+![EditCompanyDemo](images/EditCompanyDemo.png)  
+:information_source: Note the successful command execution message in the system feedback box upon successful edit of the company.
+
+Also, you can change the status of your internship application and set the reminder for the interview date by making this entry:
+2. [Edit the role](#c-edit-r) from a given company.  
+![EditRoleDemo](images/EditRoleDemo.png)  
+:information_source: Note the successful command execution message in the system feedback box upon successful edit of the role.  
+:information_source: Note the change in color of the [role tag](#tags).
+
+## Use Case #4: Setting reminder window for reminder list <a id="usecase4"></a>
+Assuming that the reminder date from [USE_CASE_3](#usecase3) is within a week from today, upon the next opening of Tinner, you will be able to see a reminder pop-out box like this:   
+![ReminderDemo](images/ReminderDemo.png)  
+
+If you do not see the intended reminder, it is likely that the reminder date is not within the [reminder window](#c-setWindow-c).  
+Thus, you can increase the reminder window by making this entry:  
+1. [Set the reminder window](#c-setWindow-c) for the reminder list.  
+![WindowrDemo](images/WindowDemo.png)    
+:information_source: Note the successful command execution message in the system feedback box upon successful change of reminder window.
```

Comment:
These can be wrapped in a div to create a blue background which is more obvious to users. Refer to the AB3 user guide for the example.